src/App.js

```diff
@@ -0,0 +1,274 @@
+// de radio button werkt nog niet goed, je kan nu wel togglen, maar hij geeft als waarde in de formstate alleen maar terug true en niet of dat overdag of 's avonds is.
+
+// de reset button werkt nog niet; ik weet niet hoe ik alles weer op 0 krijg of alles weghaal
+
+import React, {useState} from 'react';
+import './App.css';
+import Fruit from "./components/Fruit";
+import Button from "./components/Button";
+import Label from "./components/LabelInput";
+
+function App() {
+
+    const [strawberry, setStrawberry] = useState(0);
+    const [banana, setBanana] = useState(0);
+    const [kiwi, setKiwi] = useState(0);
+    const [apple, setApple] = useState(0);
+
+    function reset() {
+        setBanana(0);
+        setKiwi(0);
+        setApple(0);
+        setStrawberry(0);
+    }
+
+    // of op een andere manier:
+    // const [Fruit, setFruit] = useState(initial);
+    //
+    // const initial = {
+    //     apple: 0,
+    //     banana: 0,
+    //     kiwi: 0,
+    //     strawberry: 0
+    // }
+    //
+    // function reset(fruitType) {
+    //     setFruit(initial);
+    // }
+    //
+    // function setFruits(fruitType, value) {
+    //     if (Fruit[fruitType] >= 0) {
+    //         setFruit(Fruit =>
+    //             ({...Fruit, [fruitType]: Fruit[fruitType] + value}));
+    //     }
+    // }
+    // return (
+    //     <>
+    //         <h1>Appels</h1>
+    //         <div className="total">
+    //             <Counter
+    //                 type="button"
+    //                 id="counter"
+    //                 onClick={() => setFruits('apple', -1)}
+    //                 disabled={Fruit.apple<=0}
+    //             >
+
+
+
+    const [formState, setFormState] = useState({
+        firstName: "",
+        lastName: "",
+        age: 0,
+        postalCode: "",
+        frequency: "iedere week",
+        radio: false,
+        comments: "",
+        conditions: false
+    })
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        console.log(formState);
+        console.log("Strawberries: " + strawberry);
+        console.log("Bananas: " + banana);
+        console.log("Kiwis: " + kiwi);
+        console.log("Apples: " + apple);
+    }
+
+    function handleFormChange(e) {
+        const changedFieldName = e.target.name;
+        const newValue = e.target.type === "checkbox" ? e.target.checked : e.target.value;
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue,
+        });
+    }
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+            <div className="total">
+                <div className="fruits">
+                    <h2>Jouw fruit</h2>
+                    <article>
```

Comment:
Als om elk Fruit-component <article> moet staan, dan zou ik het in het component zelf opnemen